examples/commonvoice/fr/README.md

```diff
@@ -0,0 +1,16 @@
+# Performance Record
+# Should be installed ffmpeg , pandas !!!
```

Comment:
This line is just a note, not a header, right?

Reply:
Yes